src/core/ViewabilityTracker.ts

```diff
@@ -241,22 +244,26 @@ export default class ViewabilityTracker {
                                         newEngagedIndexes: number[]): boolean {
         const itemRect = this._layouts[index];
         let isFound = false;
+        const newIndex =
+            I18nManager.isRTL && Platform.OS === "android"
```

Comment:
```suggestion
            I18nManager.isRTL && Platform.OS === "android" && this._isHorizontal
```
To not mess up the vertical lists we need to make sure this check only applies to horizontal scrollview